mantidproject/mantid
# 28450 exclude monitors from plot

**Description of work.**
When attempting to use either bin or colorfill plot, the monitors are not shown, thus not completely dwarfing the relevant data.
  
**To test:**
- Open `ExternalData/Testing/Data/UnitTest/ILL/PANTHER/001723.nxs`
- Right-click Plot/Colorfill and check that there is viewable data, even when the color bar is linear, and that it is not to long to appear (if possible, check by comparing to the older version, which should not show anything interesting in linear color bar mode).
- Open `ExternalData/Testing/Data/UnitTest/ILL/D11/010585.nxs`
- Right-click Plot/Bin and check that the data is not dwarfed by the monitor value, and that the plot is not too slow.
- Open `ExternalData/Testing/Data/UnitTest/HET15869.raw`. Plot the colorfill, and check that the first lines are white. The scale should go up to 1e+8.



Partially fixes #28450 

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
